src/components/Game.js

```diff
@@ -25,11 +53,11 @@ const Game = () => {
         { exampleFormat }
       </p>
 
-      <RecentSubmission />
-
-      <PlayerSubmissionForm />
+      { isSubmitted || poemLines.length < 1 ? '': <RecentSubmission submission={poemLines.length > 0 ? poemLines[poemLines.length - 1] : '' } /> }
 
-      <FinalPoem />
+      { isSubmitted ? '': <PlayerSubmissionForm fields={FIELDS} index={currentPlayer} sendSubmission={sendSubmission} /> }
```

Comment:
Just note that the prop`index` is marked as required in the prop types but you're not sending it,  that's why you're getting a warning.  

You can adjust the prop types if needed.